to-do-list/test.js

```diff
@@ -24,33 +24,19 @@ function deleteListItem() {
 // add items
 itemLink.addEventListener('click', function () {
   let newItemValue = newItem.value;
+  let addTodoItem = document.createElement('li')
+  addTodoItem.innerHTML = '<input type="checkbox" class="doneCheck"><label>' + newItemValue + '</label> <a href="#" class="deleteItem">Delete this item</a>';
+  todoList.appendChild(addTodoItem);
+  newItem.value = '';
 
-  for (i = 0; i < 1; i++) {
-    let addTodoItem = document.createElement('li')
-    addTodoItem.innerHTML = '<input type="checkbox" class="doneCheck"><label>'+ newItemValue +'</label> <a href="#" class="deleteItem">Delete this item</a>';
-    todoList.appendChild(addTodoItem);
-    console.log('Added Item');
+  console.log('Added Item');
 
-    //mark as done
-    let doneItem = addTodoItem.getElementsByClassName('doneCheck')[0];
-    doneItem.addEventListener('change', markAsDone);
-    
+  //mark as done
+  let doneItem = addTodoItem.querySelectorAll('input.doneCheck')[0];
```

Comment:
You can use `querySelector('input.doneCheck')` here. It'll give you the same result.